tests/manual-test-cases/Group18-VIC-UI/setup-testbed.robot

```diff
@@ -236,7 +236,7 @@ Deploy Nimbus ESXi Server For NGC Testing
     [Return]  ${user}-${name}  ${ip}
 
 Deploy Nimbus vCenter Server For NGC Testing
```

Comment:
This also appears to be just the same as our deploy nimbus vcenter with the --useQaNgc flag, which can be pushed into our keyword by overloading the name field.